egs/wsj/s5/steps/libs/nnet3/train/common.py

```diff
@@ -357,6 +357,37 @@ def parse_generic_config_vars_file(var_file):
     raise Exception('Error while parsing the file {0}'.format(var_file))
 
 
+def parse_input_model(input_model):
```

Comment:
I will change it, also it is good if definition for 'parse_generic_config_vars_file' function is added.